### ISSUE TITLE
why SSID and Wifi Password?

### ISSUE THREAD

**Issue by jampy** [TuyaAPI/link#17]
If I'm not wrong, this package links a Tuya-based device via a wifi network (not Bluetooth).

That implies that the device must somehow already be connected to the wireless AP.

So, why must the same wireless secrets be provided again while linking?

**Comment by codetheweb** [comment]
You are correct that it links via Wi-Fi. However, it does so very cleverly by encoding the data as the length of UDP broadcast packets. This means your phone or whatever’s giving the credentials needs to be linked to *a* network but the device does not.
